extended/src/main/java/io/kubernetes/client/extended/generic/options/ListOptions.java

```diff
@@ -0,0 +1,71 @@
+package io.kubernetes.client.extended.generic.options;
```

Comment:
Shouldn't we use the generated class?

Reply:
there is a new class named `ListOption` introduced by this pull. the point of defining option classes here is to provide the developers a better programming interface, which looks making more logical sense comparing to those generated long-list parameters calls. in golang, we have similar option structs defined over the actual API parameters, developer works on that logical options and the SDK translates them to api interface options.. does that make sense to you?